tools/ktx/command_create.cpp

```diff
@@ -2105,22 +2127,57 @@ void CommandCreate::checkSpecsMatch(const ImageInput& currentFile, const ImageSp
     const FormatDescriptor& currentFormat = currentFile.spec().format();
 
     if (currentFormat.transfer() != firstFormat.transfer()) {
-        fatal(rc::INVALID_FILE, "Input image \"{}\" has different transfer function ({}) than preceding image(s) ({}).",
-            currentFile.filename(), toString(currentFormat.transfer()), toString(firstFormat.transfer()));
+        if (options.assignOETF.has_value()) {
+            warning("Input image \"{}\" has different transfer function ({}) than the first image ({})"
+                " but will be threaded identically as specified by the --assign-oetf option.",
+                currentFile.filename(), toString(currentFormat.transfer()), toString(firstFormat.transfer()));
+        } else if (options.convertOETF.has_value()) {
+            warning("Input image \"{}\" has different transfer function ({}) than the first image ({})"
+                " and thus will go through different transfer function conversion to the target transfer"
+                " function specified by the --convert-oetf option.",
+                currentFile.filename(), toString(currentFormat.transfer()), toString(firstFormat.transfer()));
+        } else {
+            fatal(rc::INVALID_FILE, "Input image \"{}\" has different transfer function ({}) than the first image ({}).",
```

Comment:
On this and the error messages for gamma and primaries add a note to the message to the effect of "Use --assign-XXX or --convert-XXX to specify handling and stop this error."